### ISSUE TITLE
Unable to GSM signal (2G only) 

### ISSUE THREAD

**Issue by raghunandanjv** [CyanogenDefy/android_device_motorola_jordan#22]
I have selected the baseband version to India(Last option in baseband switcher) and If i select 2G only option, the GSM signal becomes completely blank. If I un-select the 2G only option and use GSM/WCDMA it works(after i search and select Airtel). I am in India and my service provider is Airtel. Please let me know what else I should do. I am very happy to have 2.3.4 but if does not get me calling facility then I need to go back to 2.2.2.

Thanks,
Raghu

**Comment by charlie2alpha** [comment]
I am not sure what version you are using, but I encountered the reverse problem with the versions using the 179 kernel. Using the same baseband as before, I cannot connect to 3G reliably. 
I solved the problem for now by selecting a baseband with the same frequency ranges (900Mhz/2100Mhz in my case) but from a different ROM/country.

In more detail, the original ROM for my country is the CEE (Central & Easter Europe) so I was using the Central Europe baseband. That worked fine with the 177 kernel, but looses 3G with the 179. I switched to RT Europe 3.4.2-175 and 3G woks again. 

I request in the XDA forum if someone can find the CEE 179 baseband to try.
